libs/ui-components/src/components/Device/EditDeviceWizard/steps/DeviceTemplateStep.tsx

```diff
@@ -1,28 +1,37 @@
 import * as React from 'react';
-
-import { Form, FormGroup, Grid } from '@patternfly/react-core';
+import { Alert, Form, FormGroup, Grid } from '@patternfly/react-core';
 import { FormikErrors, useFormikContext } from 'formik';
+import { Trans } from 'react-i18next';
 
 import { useTranslation } from '../../../../hooks/useTranslation';
 import WithHelperText from '../../../common/WithHelperText';
 import TextField from '../../../form/TextField';
-
-import ConfigTemplateForm from './ConfigTemplateForm';
 import { DeviceSpecConfigFormValues } from '../types';
+import ConfigTemplateForm from './ConfigTemplateForm';
 
 export const deviceTemplateStepId = 'device-template';
 
 export const isDeviceTemplateStepValid = (errors: FormikErrors<DeviceSpecConfigFormValues>) => {
   return !errors.osImage && !errors.configTemplates;
 };
 
-const DeviceTemplateStep = () => {
+const templateCode = 'device.metadata.labels[key]';
```

Comment:
```suggestion
const templateCode = '{{ device.metadata.labels[key] }}';
```

Reply:
Good catch, I removed the brackets unvoluntirely.